temporal-test-server/src/main/java/io/temporal/internal/testservice/TestNexusEndpointStore.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2024 Temporal Technologies, Inc. All Rights Reserved.
```

Comment:
Not sure why CI is complaining about the license. All I did was copy paste, remove Amazon and Uber, and update the date. But if I was supposed to leave those alone, lmk.

Reply:
You just need to run ` ./gradlew licenseFormat`